tensorflow_2/README.md

```diff
@@ -0,0 +1,45 @@
+# Samples for Notebook to Kubeflow Deployment using TensorFlow 2.0 Keras
+This directory aims at building Kubeflow use case samples with Tensorflow 2.0 Keras training code demonstrating 'Customer User Journey' (CUJ) in the process.
+
```

Comment:
Should add prerequistes, system requirements, etc.

Reply:
Mentioned the minimum requirements and gave pointers to more details.